includes/handler/class-quote-request.php

```diff
@@ -0,0 +1,235 @@
+<?php
+/**
+ * Handler for QuoteRequest activities.
+ *
+ * @package Activitypub
+ */
+
+namespace Activitypub\Handler;
+
+use Activitypub\Activity\Activity;
+use Activitypub\Collection\Actors;
+use Activitypub\Collection\Followers;
+use Activitypub\Collection\Remote_Actors;
+
+use function Activitypub\add_to_outbox;
+use function Activitypub\object_to_uri;
+
+/**
+ * Handler for QuoteRequest activities.
+ *
+ * @see https://codeberg.org/fediverse/fep/src/branch/main/fep/044f/fep-044f.md
+ */
+class Quote_Request {
+	/**
+	 * Initialize the class, registering WordPress hooks.
+	 */
+	public static function init() {
+		\add_action( 'activitypub_inbox_quoterequest', array( self::class, 'handle_quote_request' ), 10, 2 );
+		\add_action( 'activitypub_rest_inbox_disallowed', array( self::class, 'handle_blocked_request' ), 10, 3 );
+
+		\add_filter( 'activitypub_validate_object', array( self::class, 'validate_object' ), 10, 3 );
+	}
+
+	/**
+	 * Handle QuoteRequest activities.
+	 *
+	 * @param array $activity The activity object.
+	 * @param int   $user_id  The user ID.
+	 */
+	public static function handle_quote_request( $activity, $user_id ) {
+		$state   = true;
+		$post_id = \url_to_postid( object_to_uri( $activity['object'] ) );
+
+		if ( ! $post_id ) {
+			self::queue_reject( $activity, $user_id );
+			return;
+		}
+
+		$content_policy = \get_post_meta( $post_id, 'activitypub_interaction_policy_quote', true );
+
+		switch ( $content_policy ) {
+			case ACTIVITYPUB_INTERACTION_POLICY_ME:
+				self::queue_reject( $activity, $user_id, $post_id );
```

Comment:
```suggestion
				self::queue_reject( $activity, $user_id );
```
`queue_reject()` only accepts two parameters.